pyk/src/pyk/kast.py

```diff
@@ -276,23 +276,23 @@ def match(self, term: KInner) -> Optional[Subst]:
 @dataclass(frozen=True)
 class KToken(KInner):
     token: str
-    sort: str
+    sort: KSort
 
-    def __init__(self, token: str, sort: str):
+    def __init__(self, token: str, sort: KSort):
```

Comment:
I would like to have this constructor too as well, actually. Especially since we'll be adding parametric sorts later, and we want to make it easier for users to specify non-parametric sorts/productions without having to do this.

Reply:
Done.